assets/dash.js

```diff
@@ -348,3 +348,5 @@ const myForm = document.getElementById('dash-nl');
 		}
 	await handleSubmit()
 	});
+
+*/
```

Comment:
Please add a newline at the end of the file here, then this can be merged :+1:

Reply:
thank you! done